keyspace.go

```diff
@@ -121,6 +121,39 @@ func (k *k) MultiTimeSeriesTable(name, indexField, timeField, idField string, bu
 	}
 }
 
+func (k *k) FlakeSeriesTable(name string, bucketSize time.Duration, row interface{}) FlakeSeriesTable {
+	m, ok := toMap(row)
+	if !ok {
+		panic("Unrecognized row type")
+	}
+	m[flakeTimestampFieldName] = time.Now()
+	m[bucketFieldName] = time.Now()
+	return &flakeSeriesT{
+		Table: k.NewTable(fmt.Sprintf("%s_flakeSeries", name), row, m, Keys{
```

Comment:
As with the multi flake series, I think the name should be `<name>_flakeSeries_<idField>`